var/spack/repos/builtin/packages/gdl/package.py

```diff
@@ -14,8 +14,9 @@ class Gdl(CMakePackage):
     """
 
     homepage = "https://github.com/gnudatalanguage/gdl"
-    url      = "https://github.com/gnudatalanguage/gdl/archive/v0.9.8.tar.gz"
+    url      = "https://github.com/gnudatalanguage/gdl/archive/v0.9.9.tar.gz"
```

Comment:
Please add the comment somewhere:
# This package has way too many dependencies for me to test PROJ.6 support, so I'm going to explicitly require # proj@:5 until someone has the time to make sure it works with PROJ.6.